src-docs/src/views/code/code_block.js

```diff
@@ -15,7 +15,7 @@ export default () => (
     <EuiSpacer />
 
     <EuiCodeBlock
-      language="jsx"
+      language="nonsense"
```

Comment:
Although we now have a fallback to an unsupported language I think it's better to keep the old example:
```suggestion
      language="jsx"
```

This way we make sure consumers follow and copy-paste the best practices.

Reply:
Yep, I should have noted that I will revert the "revert me" commit.